plugins/git/plugin.zsh

```diff
@@ -172,13 +176,17 @@ geometry_prompt_git_render() {
     fi
   fi
 
+  if git rev-parse --quiet --verify refs/stash >/dev/null; then
+      stashes="$GEOMETRY_GIT_STASHES";
+  fi
+
   local render="$(prompt_geometry_git_symbol)"
 
   if [[ -n $render ]]; then
     render+=" "
   fi
 
-  render+="$(prompt_geometry_git_branch) ${conflicts}${GEOMETRY_GIT_SEPARATOR}${time} $(prompt_geometry_git_status)"
+  render+="$(prompt_geometry_git_branch) ${conflicts}${GEOMETRY_GIT_SEPARATOR}${time} $(prompt_geometry_git_status) ${stashes}"
```

Comment:
I'd like to see what this looks like before the status, with the other 'optional' parts.
Also we should include the space / GEOMETRY_GIT_SEPARATOR as part of the `stashes=` assignment so if stashes is unassigned we don't have a trailing space.